.eslintrc.js

```diff
@@ -9,10 +9,13 @@ module.exports = {
     'node': true
   },
   'extends': 'eslint:recommended',
```

Comment:
What do you think about adding recommended for `@typescript-eslint` as well? See: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs.

Reply:
We also might consider https://prettier.io/ to help us with formatting. It is highly opinionated but it is a good thing. Of course this is not in the scope of this PR.

Reply:
Good idea

Reply:
Oh dear, 275 violations :see_no_evil:

Reply:
Only 3 error but I don't like warnings.

Reply:
:-( So up to you

Reply:
I reduced it to 105 warnings.
Most of it is either `userProfile`/`preferences` or `(<any>MessageType).`
Both could be solved by generating the types from the schema.

Reply:
The `MessageType` is now generated.
12 warnings remaining :sunglasses:

Reply:
0 warnings remaining :raised_hands: